### ISSUE TITLE
Read-only state "xyz" has been written without ack-flag with value

### ISSUE THREAD

**Issue by jpawlowski** [ioBroker/ioBroker.cloud#353]
Die State Objekte werden vom Adapter als Readonly / `write=false` definiert. Das ist sinnvoll, allerdings passen die dazugehörigen `setState()` Befehle nicht. Dort wird das `ack` Flag nicht auf `true` gesetzt:

https://github.com/ioBroker/ioBroker.cloud/blob/0b4e7b738b2e1ca586aa89cfe6ba8f77f4f881db/main.js#L847

Aus diesem Grund wirft der JS-Controller korrekterweise eine Warnmeldung im Log:

```
Read-only state "cloud.0.services.custom_xyz" has been written without ack-flag with value "this is my value"
```

Daher wäre es prima, wenn die `setState()` Befehle entsprechend mit `ack=true` ausgeführt würden.

**Comment by Apollon77** [comment]
Yes I will fix this, BUT "services" vis pro cloud are kind of deprecated and should be moved to iot adapter!

**Comment by jpawlowski** [comment]
> "services" vis pro cloud are kind of deprecated and should be moved to iot adapter!

That is good to know, thank you! I didn't know this. Was there any official info about it? Asking bc I'm currently integrating some API handling into the ioBroker.residents adapter using `ioBroker.simple-api`, `ioBroker.cloud`, and apparently now also `ioBroker.iot` adapters.

**Comment by jpawlowski** [comment]
still a topic for discussion/resolution

**Comment by tommyMX** [comment]
Any news ?

**Comment by tommyMX** [comment]
Read-only state "iot.0.services.custom_location.home" has been written without ack-flag with value "false"
